### PR TITLE
[HiTL] Added a simple example to demonstrate HiTL lib usage

### PR DESCRIPTION
**Pull request by snyxan**
# Description

Implemented a simple example to demonstrate how to use HiTL library. The file/fn name and comments should be self-explanatory. 

Read #792 will help understand this better.

Some of the comments in #721 are addressed in this PR as well.

## Type of change

Please check the options that are relevant.

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] Proposes a change (non-breaking change that isn't necessarily a bug)
- [ ] Refactor
- [x] New feature (non-breaking change that adds a new functionality)
- [ ] Breaking change (fix or feature that would break some existing functionality downstream)
- [ ] This is a unit test
- [ ] Documentation only change
- [ ] Datasets Release
- [ ] Models Release

## Type of requested review

- [ ] I want a thorough review of the implementation.
- [x] I want a high level review. 
- [ ] I want a deep design review.

# Testing

Tested locally and passed.

# Checklist:

- [x] I have performed manual end-to-end testing of the feature in my environment.
- [x] I have added Docstrings and comments to the code.
- [ ] I have made changes to existing documentation where needed.
- [ ] I have added tests that show that the PR is functional.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] I have added relevant collaborators to review the PR before merge.
- [ ] [Polymetis only] I have verified that all scripts in `tests/scripts` runs on hardware.

### CODE REVIEW

File: droidlet/tools/hitl/simple_hitl_example.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+import time
+
+from typing import List
+
+from .data_generator import DataGenerator
+from .job_listener import JobListener
+from .task_runner import TaskRunner
+
+
+WORKDIR = "/private/home/yuxuans/Workspace/tmp/droidlet/droidlet/tools/hitl/tmp/"
```
Comment (ethan-carlson) [inline droidlet/tools/hitl/simple_hitl_example.py:24]: Might not want your actual filepath committed to main

**Review by ethan-carlson — APPROVED** [review]

File: droidlet/tools/hitl/simple_hitl_example.py (hunk 1 above)
Comment (kavyasrinet) [reply droidlet/tools/hitl/simple_hitl_example.py:24]: Agree with this, lets not add hardcoded paths

**Review by kavyasrinet — APPROVED** [review]
Approving but lets remove the hard coded path

File: droidlet/tools/hitl/simple_hitl_example.py (hunk 1 above)
Comment (snyxan) [reply droidlet/tools/hitl/simple_hitl_example.py]: resolved.

**Pull request merged**